anchore-policy-validator/README.md

```diff
@@ -0,0 +1,42 @@
+# Anchore policy validator
+
+This chart deploy an admission-server that is used as a ValidatingWebhook in a k8s cluster. If it's working, kubernetes will send requst to admission server when a Pod creation is initiated. The server check image, which is defined in PodSpec, against configured Anchore-engine API. If API response that image not valid according to defined policy, k8s will reject Pod creation request.
```

Comment:
`deploy` -> `deploys`
`requst` -> `requests`
`to admission` -> `to the admission`
`check image` -> `checks the image`
`If API response...` -> `If the API responds with an error, that the image is not valid according to defined policy, k8s will reject the Pod creation request.`